app/javascript/packages/phone-input/index.spec.ts

```diff
@@ -207,8 +210,10 @@ describe('PhoneInput', () => {
 
       const phoneNumber = getByLabelText(input, 'Phone number') as HTMLInputElement;
 
-      await userEvent.type(phoneNumber, '513-555-1234');
```

Comment:
I'm curious why we needed to change the phone number example here. I'd expect this number to be invalid for the given context.

Reply:
Now that I see this again, it looks like it comes down to `isValidNumberForRegion` vs. `isValidNumber`.